src/framework/handlers/basis.js

```diff
@@ -87,7 +87,15 @@ const prepareWorkerModules = (config, callback) => {
                 }
             })
             .catch((err) => {
-                callback(err, null);
+                Debug.warn(`compile() failed for ${config.wasmUrl} (${err}), using fallback script.`);
+
+                http.get(config.fallbackUrl, options, (err, response) => {
```

Comment:
Is the fallback script mandatory? I thought some WASM modules out there didn't have a fallback. So assuming it's _not_ guaranteed to be present, this should probably check it's set?

Reply:
Oh, right yeah, this is for Basis specifically. I thought it was a generic code path for loading modules.